package.json

```diff
@@ -59,7 +60,8 @@
   },
   "repository": "guardian/cdk-cli",
   "scripts": {
-    "prepack": "rm -rf lib && tsc -b && oclif-dev readme",
+    "prepack": "rm -rf lib && tsc -b && yarn copy-files && oclif-dev readme",
+    "copy-files": "copyfiles -a -u 1 \"./src/template/**/*\" lib",
```

Comment:
Not sure if this is the best approach to including static files. Any other ideas?